packages/pds/tests/event-stream/__snapshots__/sync.test.ts.snap

```diff
@@ -186,14 +186,28 @@ Array [
       "uri": "record(2)",
       "viewer": Object {},
     },
+    "reason": Object {
```

Comment:
did these snaps change so much cause of the new `sortBy`?

Reply:
Yeah, that's right.  I think the current behavior is more correct because the feed usually shouldn't be reordered based on the updated `indexedAt` time (at least when folks are playing nice with their self-reported `createdAt` times).